packages/dds/test-dds-utils/src/gcTestRunner.ts

```diff
@@ -36,6 +38,11 @@ export interface IGCTestProvider {
 	addNestedHandles(): Promise<void>;
 }
 
+// eslint-disable-next-line jsdoc/require-description
```

Comment:
Is this needed?

Reply:
![image](https://github.com/microsoft/FluidFramework/assets/111468570/fa884868-1082-4b36-af70-014c8f2be02d)

Explained above. I could also leave a brief description for the functions (`replayTest` & `runGCTests`) out of my understanding (or could leave `TODO`).